mautic/mautic
# [UX] Disable permission tab when "Has full system access?" is set to Yes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses a usability issue in the user interface regarding the permissions tab. When the "Has full system access?" option is set to Yes, it becomes redundant to specify additional permissions, as the user already has complete access.

By disabling the permissions tab in this scenario, we streamline the user experience, reducing confusion and unnecessary clicks. This change benefits administrators and users alike by clarifying access levels. Overall, it enhances the interface's intuitiveness and efficiency.







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Roles page, create a new role, and open the permissions tab. Go back to the first tab, toggle "Has full system access," and try to open the permissions tab again. It should fail. Go back to the first tab, toggle the button again, and test the permissions tab to ensure it succeeds.
